### PR TITLE
Added Logger

### PR DESCRIPTION
**Pull request by SheraliErkayev**

### CODE REVIEW

File: TestHotel/TestHotel.DataAccess/Repository/Repositories/BillRepository.cs [hunk 1]
```diff
@@ -25,10 +23,12 @@ public async Task<int> AddBillAsync(Bill bill)
             {
                 _context.Bills.Add(bill);
                 await _context.SaveChangesAsync();
+                _logger.LogInformation("AddBillAsync() Chaqirildi");
                 return bill.InvoiceNumber;
             }
-            catch 
+            catch
             {
+                _logger.LogError("AddBillAsync() Qo'shilmadi");
```
Comment (JmlKhon) [inline TestHotel/TestHotel.DataAccess/Repository/Repositories/BillRepository.cs]: Log messagelar aniqroq yozish kerak. 
AddBillAsync() Chaqirildi => Bill muvaffaqiyatli qo'shildi.
AddBillAsync() Qo'shilmadi => bill yaratilishida xatolik yuzaga keldi

**Review by JmlKhon — CHANGES_REQUESTED** [review]

File: TestHotel/TestHotel.DataAccess/Repository/Repositories/BillRepository.cs (hunk 1 above)
Comment (JmlKhon) [reply TestHotel/TestHotel.DataAccess/Repository/Repositories/BillRepository.cs]: hamma loggerlarni shunaq qb togirlash kerak. Log qiganda methodni nomi chiqadi qayerdan log qilinilayotganligi shuchun method nomi kerak emas

Comment (SheraliErkayev) [reply TestHotel/TestHotel.DataAccess/Repository/Repositories/BillRepository.cs]: ok

File: TestHotel/TestHotel.API/appsettings.json [hunk 2]
```diff
@@ -7,5 +7,21 @@
...
+        "Using": [ "Serilog.Sinks.File" ],
+        "MinimumLevel": {
+            "Default": "Information"
+        },
+        "WriteTo": [
+            {
+                "Name": "File",
+                "Args": {
+                    "path": "../logs/webapi-.log",
+                    "rollingInterval": "Day",
```
Comment (JmlKhon) [inline TestHotel/TestHotel.API/appsettings.json]: make it 7 days

File: TestHotel/TestHotel.API/appsettings.json [hunk 3]
```diff
@@ -7,5 +7,21 @@
             "Default": "Information",
             "Microsoft.AspNetCore": "Warning"
         }
+    },
+    "Serilog": {
+        "Using": [ "Serilog.Sinks.File" ],
+        "MinimumLevel": {
+            "Default": "Information"
```
Comment (JmlKhon) [inline TestHotel/TestHotel.API/appsettings.json]: Error

**Review by JmlKhon — CHANGES_REQUESTED** [review]

File: TestHotel/TestHotel.DataAccess/Repository/Repositories/RoomTypeRepository.cs [hunk 4]
```diff
@@ -55,12 +57,14 @@ public async Task<RoomType> GetRoomTypeByIdAsync(RoomTypes roomTypes)
         {
             try
             {
+                _logger.LogInformation("RoomTypeById muvaffaqiyatli topildi");
                 return await _context.RoomTypes
                     .Include(u => u.Room)
                     .FirstOrDefaultAsync(u => u.RoomTypes == roomTypes);
```
Comment (MirazamTUT) [inline TestHotel/TestHotel.DataAccess/Repository/Repositories/RoomTypeRepository.cs:63]: RoomTypeRepository dagi va IRoomTypeRepository dagi GetRoomTypeById ni int id oladigan va 
(u => u.RoomTypeId == id) bo'lishi kerak

**Review by MirazamTUT — CHANGES_REQUESTED** [review]

File: TestHotel/TestHotel.DataAccess/Repository/Repositories/RoomTypeRepository.cs (hunk 4 above)
Comment (JmlKhon) [reply TestHotel/TestHotel.DataAccess/Repository/Repositories/RoomTypeRepository.cs:63]: it's out of scope of this ticket. Please, Create a defect ticket for it. @MirazamTUT

Comment (MirazamTUT) [reply TestHotel/TestHotel.DataAccess/Repository/Repositories/RoomTypeRepository.cs:63]: hop

**Review by MirazamTUT — APPROVED** [review]

**Review by isfandiyoro5 — APPROVED** [review]

**Review by mrIminjonov — APPROVED** [review]

**Review by Bekhzod7799 — APPROVED** [review]

**Review by MirazamTUT — APPROVED** [review]

**Review by isfandiyoro5 — APPROVED** [review]

File: TestHotel/TestHotel.API/appsettings.json (hunk 2 above)
Comment (isfandiyoro5) [reply TestHotel/TestHotel.API/appsettings.json]: done

**Review by isfandiyoro5 — APPROVED** [review]

**Review by mrIminjonov — APPROVED** [review]

**Review by JmlKhon — APPROVED** [review]

**Pull request merged**